fenix/app/src/main/java/org/mozilla/fenix/debugsettings/navigation/FenixDebugDrawerDestinations.kt

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
nit: Can we drop the `Fenix` prefix? Perhaps this could also be called `DebugDrawerNavigation`

Reply:
If it becomes `DebugDrawerNavigation` maybe `DebugDrawerDestination` & this file can be consolidated